src/Styles/Form.module.css

```diff
@@ -1,6 +1,7 @@
 .form {
```

Comment:
- [ ] Try to duplicate the Zeplin design, it should be identical to your app, kindly use the same padding/margin, font size, and color

|Your design|Zeplin design|
|-|-|
|![Screenshot_20220609_193041](https://user-images.githubusercontent.com/25227022/172920344-629c7ac6-6632-45aa-b226-a05d38f78e38.png)|![Screenshot_20220609_193901](https://user-images.githubusercontent.com/25227022/172920428-ae54d0ed-9287-4d96-b174-7bf3339c277b.png)|